.eslintrc.js

```diff
@@ -70,9 +73,14 @@ module.exports = {
     'no-multi-spaces': 'warn',
     'arrow-spacing': ['error', { before: true, after: true }],
     'space-in-parens': 'warn',
-    'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
+    'react/jsx-filename-extension': [
+      2,
+      { extensions: ['.js', '.jsx', '.ts', '.tsx'] },
+    ],
     'prefer-destructuring': 'off',
     'import/no-unresolved': 'off',
+    'react/static-property-placement': 'off',
+    'react/no-unused-class-component-methods': 'off',
```

Comment:
@athottungal this is a react rule that essentially disallows public methods, including getters and setters

since CAM has 100s of instances of this (it's built like a traditional JS Class as opposed to using React best practices), I don't think having 100s of warnings is actually useful to us